eng/common-tests/ci.yml

```diff
@@ -0,0 +1,28 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - eng/common/scripts/*
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - eng/common/scripts/*
+
+extends:
+  template: /eng/common/pipelines/templates/stages/archetype-sdk-tool-pwsh.yml
```

Comment:
@konrad-jamrozik it would be good to hook up the matrix tests you are moving in https://github.com/Azure/azure-sdk-tools/pull/5027 to this pipeline once this goes in.

Reply:
@weshaggard addressed by:
- #5095